src/brevitas/core/scaling/standalone.py

```diff
@@ -374,6 +374,12 @@ def __init__(
         self.restrict_inplace_preprocess = restrict_scaling_impl.restrict_init_inplace_module()
         self.restrict_scaling_pre = restrict_scaling_impl.restrict_init_module()
         self.restrict_threshold_pre = restrict_threshold_impl.restrict_init_module()
+        self.init_done: bool = brevitas.jit.Attribute(False, bool)
+
+    def init_scale(self):
+        self.restrict_inplace_preprocess(self.buffer)
+        inplace_tensor_mul(self.value.detach(), self.buffer)
+        self.counter = self.counter + 1
```

Comment:
Does this need to set `self.init_done = True`?

Reply:
Init done is not needed. I added it in this PR but it's redundant with the last changes